### ISSUE TITLE
German Audio check of Parser doesnt cover majority of releases

### ISSUE THREAD

**Issue by ToasterUwU** [Sonarr/Sonarr#7635]
### Is there an existing issue for this?

- [x] I have searched the existing open and closed issues

### Current Behavior

Currently "GER" and "DE" arent recognized as German. 

### Expected Behavior

Since the current regex only detects a fraction of the real life releases properly, i would expect it to be changed so that it does detect "GER" and "DE" as German.

### Environment

```markdown
- OS: NixOS 24.11
- Sonarr: 4.0.12.2823
- Docker Install: Yes
- Using Reverse Proxy: No
- Browser: 
- Database: Sqlite 3.48.0
```

### What branch are you running?

Main

### Trace Logs?

Got none for you, not an error.

### Anything else?

This issue was also reported on Radarr and has been fixed via this PR: https://github.com/Radarr/Radarr/pull/10819

So it should be as simple as cherry picking those changes over to here.

**Comment by markus101** [comment]
Example release names for TV are required to consider porting over that change.

**Comment by ToasterUwU** [comment]
Ok, i assume just here as a comment is fine? Correct me if im wrong.

> Show Name S01 Eng Fre Ger Ita Spa Cze Jpn 2160p BluRay Remux DV HDR HEVC Atmos SGF
> Show Name.S01.Eng.Fre.Ger.Ita.Por.Spa.2160p.WEBMux.DV.HDR.HEVC.Atmos-SGF
> Show.Name.S02E10.Episode.Title.German.DL.BD.x264-TVS
> Show Name S01 Eng Fre Ger Ita Por Spa 2160p WEBMux HDR HEVC DDP SGF
> Show Name S01 KOREAN ENG FRA GER ITA SPA MULTI 2160p NF WEB DL DDP5 1 DV HDR x265 Atmos MassModz
> Show.Name.S02E09.EpisodeName.German.DL.BD.x264-TVS

I personally couldnt find a example for "DE", im sure i have seen it before a few times, but im not sure if its was exclusively for Movies (which i can find examples for)

Let me know if you need something else!
